### ISSUE TITLE
[v0.4.4-rc.1 - K8s] [Import/Export] Import fails after restarting flowbroker

### ISSUE THREAD

**Issue by betefaber** [dojot/dojot#1823]
Affected baseline **v0.4.4-rc.1**, kubernetes environment

**Steps to reproduce the problem**:

- Create templates, devices, flows and remote nodes
- Export the data
- Restart the dojot
- Import the data that was exported

```
<20:08:00 30/10/2020> -- DEBUG: Will import data.
<20:08:00 30/10/2020> -- DEBUG: Will call delete http://flowbroker:80/v1/flow
<20:08:00 30/10/2020> -- DEBUG: Flows deleted.
<20:08:00 30/10/2020> -- DEBUG: Will call delete http://flowbroker:80/v1/node
<20:08:00 30/10/2020> -- DEBUG: Nodes deleted.
<20:08:00 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/node
<20:08:08 30/10/2020> -- DEBUG: Nodes imported.
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:08:08 30/10/2020> -- DEBUG: Flows imported.
<20:08:08 30/10/2020> -- DEBUG: Will call http://device-manager:5000/import
<20:08:11 30/10/2020> -- DEBUG: Devices and templates imported.
<20:08:11 30/10/2020> -- DEBUG: Will call delete http://cron:5000/cron/v1/jobs
<20:08:11 30/10/2020> -- DEBUG: Cron jobs deleted
<20:08:11 30/10/2020> -- DEBUG: Cron jobs imported
```

----
- Repeat the import

```
<20:09:23 30/10/2020> -- DEBUG: Will import data.
<20:09:23 30/10/2020> -- DEBUG: Will call delete http://flowbroker:80/v1/flow
<20:09:23 30/10/2020> -- DEBUG: Flows deleted.
<20:09:23 30/10/2020> -- DEBUG: Will call delete http://flowbroker:80/v1/node
<20:09:24 30/10/2020> -- DEBUG: Nodes deleted.
<20:09:24 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/node
<20:09:26 30/10/2020> -- DEBUG: Nodes imported.
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:26 30/10/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<20:09:27 30/10/2020> -- DEBUG: Flows imported.
<20:09:27 30/10/2020> -- DEBUG: Will call http://device-manager:5000/import
<20:09:30 30/10/2020> -- DEBUG: Devices and templates imported.
<20:09:30 30/10/2020> -- DEBUG: Will call delete http://cron:5000/cron/v1/jobs
<20:09:30 30/10/2020> -- DEBUG: Cron jobs deleted
<20:09:30 30/10/2020> -- DEBUG: Cron jobs imported
```

----
- Restart flowbroker
- Repeat the import

![Screenshot_5](https://user-images.githubusercontent.com/37310063/97753790-1c4b8680-1ad5-11eb-8e1a-c17926128b63.png)

data-manager log
```
<20:11:36 30/10/2020> -- DEBUG: Will import data.
<20:11:36 30/10/2020> -- DEBUG: Will call delete http://flowbroker:80/v1/flow
<20:11:36 30/10/2020> -- DEBUG: Flows deleted.
<20:11:36 30/10/2020> -- DEBUG: Will call delete http://flowbroker:80/v1/node
<20:11:36 30/10/2020> -- DEBUG: Received error on import data, Error: Request failed with status code 500. Rejecting the request
```

flowbroker log
```
<20:11:36 30/10/2020> -- |api -- ERROR: Error: Not found remote node with id kelvin.
    at Promise (/opt/flowbroker/orchestrator/src/nodeManager.js:349:25)
    at new Promise (<anonymous>)
    at NodeManager.delRemoteNode (/opt/flowbroker/orchestrator/src/nodeManager.js:318:12)
    at nodeManager.getAll.then (/opt/flowbroker/orchestrator/src/api.js:90:33)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
```

**Comment by betefaber** [comment]
Fixed in **v0.5.0-rc.2**.

```
<21:28:44 27/11/2020> -- DEBUG: Will import data.
<21:28:44 27/11/2020> -- DEBUG: Will call delete http://flowbroker:80/v1/flow
<21:28:44 27/11/2020> -- DEBUG: Flows deleted.
<21:28:44 27/11/2020> -- DEBUG: Will call delete http://flowbroker:80/v1/node
<21:28:44 27/11/2020> -- DEBUG: Nodes deleted.
<21:28:44 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/node
<21:28:45 27/11/2020> -- DEBUG: Nodes imported.
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Will call http://flowbroker:80/v1/flow
<21:28:45 27/11/2020> -- DEBUG: Flows imported.
<21:28:45 27/11/2020> -- DEBUG: Will call http://device-manager:5000/import
<21:28:48 27/11/2020> -- DEBUG: Devices and templates imported.
<21:28:48 27/11/2020> -- DEBUG: Will call delete http://cron:5000/cron/v1/jobs
<21:28:48 27/11/2020> -- DEBUG: Cron jobs deleted
<21:28:48 27/11/2020> -- DEBUG: Cron jobs imported
```

**Comment by betefaber** [comment]
The solution in v0.5.0 must be propagated to v0.4.4.
